src/testrender/optixraytracer.cpp

```diff
@@ -58,15 +59,21 @@ OptixRaytracer::~OptixRaytracer ()
 std::string
 OptixRaytracer::load_ptx_file (string_view filename)
 {
-    std::string ptx_string;
-    std::string filepath = filename;
-    if (! OIIO::Filesystem::exists(filepath))
-        filepath = OIIO::Strutil::sprintf ("%s/%s", PTX_PATH, filename);
-    if (! OIIO::Filesystem::read_text_file (filepath, ptx_string)) {
-        errhandler().severe ("Unable to load %s", filename);
-        ptx_string.clear();
+#ifdef OSL_USE_OPTIX
```

Comment:
Do we need this guard here? There is no code enclosed that makes OptiX API calls.